kube-config-files/methode-list-mapper-service.yaml

```diff
@@ -0,0 +1,70 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name:  methode-list-mapper
+  labels:
+    name:  methode-list-mapper
```

Comment:
please use "app":  methode-list-mapper label here, since we select deployments based on this label